assets/js/pages/SapSystemDetails/sapOperations.js

```diff
@@ -0,0 +1,33 @@
+import { curry } from 'lodash';
```

Comment:
I like the idea of using it, but to be honest i had first read about it :)
It is cleaner then writing some wrappers!